include/crow/websocket.h

```diff
@@ -555,7 +603,11 @@ namespace crow
                         if (is_FIN())
```

Comment:
Maybe move this `if` to separate method?

Reply:
Can do